paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/AffirmDefinition.kt

```diff
@@ -33,7 +34,10 @@ internal object AffirmDefinition : PaymentMethodDefinition {
             lightThemeIconUrl = sharedDataSpec.selectorIcon?.lightThemePng,
             darkThemeIconUrl = sharedDataSpec.selectorIcon?.darkThemePng,
             tintIconOnSelection = false,
-            formSpec = LayoutSpec(sharedDataSpec.fields),
+            formElements = transformSpecToElements.transform(
+                specs = sharedDataSpec.fields,
+                requiresMandate = false,
```

Comment:
requires mandate is now being set in two places within the SupportedPaymentMethod constructor -- is there a way to just set it in one place?

Reply:
One idea I have is to move the requiresMandate into a separate property in `PaymentMethodDefinition`. Then we could use that in all the places. But we do need to provide it to multiple places.

We use it for knowing if we can auto select something in flow controller (before the user would launch it), as well as to know if we should display the mandate text.

That might make it a little more clear. Thoughts on doing that as a follow up? Or were you thinking of a separate problem?

Reply:
yeah my concern is about having to define it in multiple places. If it was defined as a separate property in PaymentMethodDefinition, then I think we could just define it once and then provide it where necessary, right? Doing that in a follow up sounds good to me!

Reply:
https://github.com/stripe/stripe-android/pull/8128